src/main.c

```diff
@@ -79,7 +114,13 @@ int main(int argc, char *argv[])
 
     else if (strcmp(argv[1], "run") == 0)
     {
-        return run();
+        if (argc > 3)
+        {
+            printf("%serror:%s unexpected argument '%s'\n", RED, RESET, argv[3]);
+            exit(1);
+        }
+
+        return run(argc == 2 ? argv[2] : "default.model");
```

Comment:
this is the wrong way. If `argc == 2`, `default.model` should be loaded

Reply:
true